docs/source/tutorials/index.rst

```diff
@@ -0,0 +1,308 @@
+Similarly, the ``parameters`` can be shown using:
+
+.. code-block:: console
+
+    verdi node attributes 59516
+
+    PK: 59516
+    {
+        "CONTROL": {
+            "calculation": "scf"
+        },
+        "SYSTEM": {
+            "ecutrho": 240.0,
+            "ecutwfc": 30.0
+        }
+    }
+
+Likewise, the output parameters (``output_parameters``), which are of the same type (``Dict``) of the input parameters (``parameters``) can also be shown:
+
+.. code-block:: console
+
+    verdi node attributes 59522
+
+    PK: 59522
+    {
+        "beta_real_space": false,
+        "convergence_info": {
+            "scf_conv": {
+                "convergence_achieved": true,
+                "n_scf_steps": 6,
+                "scf_error": 2.1572731407236e-07
+            }
+        },
+        "creator_name": "pwscf",
+        "creator_version": "6.6",
+        "energy": -308.19211637089,
+        ...
+        "energy_accuracy": 5.850447441879e-06,
+        "energy_accuracy_units": "eV",
+        "fermi_energy": 6.508401516556,
+        "fermi_energy_units": "eV",
+        "number_of_atoms": 2,
+        "number_of_bands": 4,
+        "number_of_bravais_symmetries": 48,
+        "number_of_electrons": 8.0,
+        "number_of_k_points": 3,
+        "number_of_species": 1,
+        "number_of_spin_components": 1,
+        "number_of_symmetries": 48,
+        "scf_iterations": 6,
+        "volume": 40.02575122515,
+        "wall_time_seconds": 1.23,
+    }
+
+The output will show a number of parameters that have been automatically parsed from the output of the ``pw.x`` calculation.
+Note that the output has been shortened for clarity and not all parsed output parameters are shown.
+
+
+.. _tutorials:pw-through-api:
+
+Running a ``pw.x`` calculation through the API
+==============================================
+
+The following minimal example shows how to run a ``pw.x`` calculation through AiiDA's Python API.
+For the purposes of this demonstration, the electronic ground state of an fcc silicon crystal structure is computed.
+
+.. tip::
+
+    The code that is shown below in snippets can be :download:`downloaded as a script <include/scripts/run_pw_basic.py>`.
+    Save the script to a file, make it executable and then simply execute it to run the example calculation.
+
+First, import the required classes and functions:
+
+.. code-block:: python
+
+    from aiida.engine import run
+    from aiida.orm import Dict, KpointsData, StructureData, load_code, load_group
+
+Then, load the code that was setup in AiiDA for ``pw.x`` and get an instance of the `process builder <https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/processes/usage.html#process-builder>`_:
+
+.. code-block:: python
+
+    # Load the code configured for ``pw.x``. Make sure to replace this string
+    # with the label that you used in the :ref:`code setup <installation:setup:code>`_.
```

Comment:
This is not actually converted into a link.

Reply:
How would you recommend we deal with this situation? I don't want to hardcode the link either.

Reply:
I just removed the link and said "you used in the code setup.". Should be clear enough